cloud/blockstore/libs/storage/disk_registry/disk_registry_state.cpp

```diff
@@ -2652,6 +2652,36 @@ auto TDiskRegistryState::CreateDiskPlacementInfo(
     };
 }
 
+auto TDiskRegistryState::GetAgentIdSuitableForLocalDiskAllocationAfterCleanup(
```

Comment:
```suggestion
auto TDiskRegistryState::GetAgentIdSuitableForLocalDiskAllocationAfterSecureErase(
```
cleanup - это другое